### PR TITLE
Update parquet.thrift

### LINKED ISSUE #21: Update parquet.thrift
The current `parquet.thrift` is outdated - we should update to the latest one (https://github.com/apache/parquet-format/blob/master/src/main/thrift/parquet.thrift) (and hopefully this won't break anything..)

### PR DESCRIPTION
**Pull request by sadikovi**
This PR updates `parquet.thrift` file to the latest version in parquet-format `f1de77d31936f4d50f1286676a0034b6339918ee` (https://github.com/apache/parquet-format/commit/f1de77d31936f4d50f1286676a0034b6339918ee). All tests pass locally after I added missing codec types LZ4 and ZSTD.

Closes #21

### CODE REVIEW

**Comment by sadikovi** [comment]
@sunchao Could you review please? Thanks!

I simply copied format to the repository. There are quite a few changes to the format itself with additions like column index, etc. For example, there is added LogicalType to replace ConvertedType. Do we need to add support for it?

**Comment by sadikovi** [comment]
We should also create issues to add those LZ4 and ZSTD support!

File: src/basic.rs [hunk 1]
```diff
@@ -225,7 +227,9 @@ impl convert::From<parquet::CompressionCodec> for Compression {
       parquet::CompressionCodec::SNAPPY => Compression::SNAPPY,
       parquet::CompressionCodec::GZIP => Compression::GZIP,
       parquet::CompressionCodec::LZO => Compression::LZO,
-      parquet::CompressionCodec::BROTLI => Compression::BROTLI
+      parquet::CompressionCodec::BROTLI => Compression::BROTLI,
+      parquet::CompressionCodec::LZ4 => Compression::LZ4,
```
Comment (sunchao) [inline src/basic.rs]: You may need to add a few trivial tests in `test_display_compression` and `test_from_compression`.

Comment (sadikovi) [reply src/basic.rs]: Will add, thanks.

**Comment by sadikovi** [comment]
@sunchao I addressed your comments. Could you have a look again? Thanks!

**Comment by sunchao** [comment]
Merged. Thanks again @sadikovi !

**Comment by sadikovi** [comment]
@sunchao thanks for the review and merge!

**Pull request merged**